.vscode/launch.json

```diff
@@ -0,0 +1,48 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "List basepaths",
+      "runtimeArgs": ["-r", "ts-node/register"],
```

Comment:
Please could you refer to the npm commands in this file eg ,`npm run list-basepaths` rather than the underlying node commands? That way implementation could change without having to update this file.

Reply:
These configurations are just to make it easier for local debugging. Do you think it's unnecessary?

Reply:
I think it's great that you're including this! I was just thinking of using (for example):
```
{
  "type": "node",
  "request": "launch",
  "name": "List basepaths",
  "runtimeArgs": ["run", "list-basepaths"],
  "cwd": "${workspaceFolder}/generator",
  "args": ["npm"]
},
```

That way the implementation of the `list-basepaths` command can change, and we don't need to remember to update this file.

Reply:
Got it, I would give it a try.